mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule RabbitMQCloudWatchExporter.Mixfile do
   def project do
     [
       app: :rabbitmq_cloudwatch_exporter,
-      version: "1.0.0",
+      version: "1.0.1",
```

Comment:
There is no need to bump the version if one of the dependencies are updated. 

Version is associated to changes within the code base and not its dependencies.

Reply:
Sorry I never touched Elixir before..